traitsui/testing/tester/locator.py

```diff
@@ -35,3 +37,15 @@ class TargetByName:
     """
     def __init__(self, name):
         self.name = name
+
+
+class WidgetType(enum.Enum):
+    """ A locator for locating nested widgets within a UI. Many editors will
+    contain many sub-widgets (e.g. a textbox, slider, tabs, buttons, etc.).
+
+    For example when working with a range editor, one could call
+    tester.find_by_name(ui, "ranged_number").locate(locator.WidgetType.textbox)
```

Comment:
I think we need double backticks here for it to be rendered as code in the API documentation.
```
  ``tester.find_by_name(ui, "ranged_number").locate(locator.WidgetType.textbox)``
```